### PR TITLE
fix: skip chrome and firefox downloads on the RUM agent Docker image

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

it skips chrome download from puppeteer install, and skips browsers download from playwright install.

## Why is it important?

Those downloads are not needed and impact the build time and the resilience of the build.

### CODE REVIEW

File: docker/rum/Dockerfile [hunk 1]
```diff
@@ -5,11 +5,28 @@ ARG RUM_AGENT_REPO=elastic/apm-agent-rum-js
...
+      g++ \
+      build-essential \
+      fonts-ipafont-gothic \
+      fonts-wqy-zenhei \
+      fonts-thai-tlwg \
+      fonts-kacst \
+      ttf-freefont \
+      --no-install-recommends
+
+RUN curl -sSfkL https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
```
Comment (kuisathaverat) [inline docker/rum/Dockerfile:25]: we split the installation of the deb packages and the chrome package to be able to reuse the deb package stage container as cache for other builds.

**Review by v1v — APPROVED** [review]

**Review by vigneshshanmugam — APPROVED** [review]

**Review by cachedout — APPROVED** [review]

**Review by mdelapenya — APPROVED** [review]

**Comment by kuisathaverat** [comment]
I am checking why the change causes the following error on the RUM agent

```
> elastic-apm-rum@0.0.0-monorepo serve /rumjs-integration-test
> node ./dev-utils/run-script.js startTestServers "packages/rum"

Running: startTestServers(packages/rum) 

[Static Server] - serving on:  packages/rum 8000
[Backend Server] - serving on:  8003
[APM Server Proxy] - For http://localhost:8200 on: 8001
Page Logs: E2E Global Configs {
  "agentConfig": {
    "serverUrl": "https://apm-server:8200",
    "serviceName": "test",
    "name": "rum"
  },
  "testConfig": {
    "serverUrl": "https://apm-server:8200",
    "mockBackendUrl": "http://localhost:8003",
    "stackVersion": ""
  },
  "useMocks": false,
  "mockApmServer": false
}
Page Logs: Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.
Page Logs: got data!
traceparent: 00-286ac3ad697892c406528f13c82e0ce1-e4dfc6b62a621f79-01
Page Logs: distributed tracing header value {"version":"00","traceId":"286ac3ad697892c406528f13c82e0ce1","parentId":"e4dfc6b62a621f79","flags":"01"}
OPTIONS http://localhost:8003/fetch 200
traceparent: 00-286ac3ad697892c406528f13c82e0ce1-15d4135b6ea719d5-01
POST http://localhost:8003/fetch 200
Page Logs: distributed tracing header value {"version":"00","traceId":"286ac3ad697892c406528f13c82e0ce1","parentId":"15d4135b6ea719d5","flags":"01"}
Error on http://localhost:8000/test/e2e/general-usecase/:
 Error: Error: timeout test error with a secret
    at ye (http://localhost:8000/test/e2e/general-usecase/app.e2e-bundle.min.js?token=secret:1:72157)
    at http://localhost:8000/test/e2e/general-usecase/app.e2e-bundle.min.js?token=secret:1:73025
OPTIONS https://apm-server:8200/intake/v2/rum/events 200
POST https://apm-server:8200/intake/v2/rum/events 202
[Intgration Tests Error] TimeoutError: Timeout exceeded while waiting for event
    at Timeout.<anonymous> (/rumjs-integration-test/node_modules/puppeteer/lib/helper.js:196:24)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
  -- ASYNC --
    at Page.<anonymous> (/rumjs-integration-test/node_modules/puppeteer/lib/helper.js:111:15)
    at runIntegrationTest (/rumjs-integration-test/dev-utils/integration-test.js:47:44)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async /rumjs-integration-test/dev-utils/test-servers.js:96:22
```

**Comment by vigneshshanmugam** [comment]
Seems like to me the browser process is not getting killed.

**Comment by kuisathaverat** [comment]
this message from the documentation seems relevant

```
NOTE Puppeteer works best with the version of Chromium it is bundled with. There is no guarantee it will work with any other version. Use executablePath option with extreme caution. If Google Chrome (rather than Chromium) is preferred, a Chrome Canary or Dev Channel build is suggested.
```

**Comment by kuisathaverat** [comment]
```
tests/agent/test_rum.py:12: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

req = <Response [500]>
endpoint = <tests.endpoint.Endpoint object at 0x7f1f1e6cacd0>

    def check_request_response(req, endpoint):
        msg = "Expected {}, got {}".format(endpoint.text, req.text)
>       assert req.text.strip() == endpoint.text, msg
E       AssertionError: Expected done, got Failed to launch chrome! spawn /opt/google/chrome-unstable EACCES
E       
E       
E       TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

tests/utils.py:38: AssertionError
```

**Comment by kuisathaverat** [comment]
I finally decided to run things step by step, and I've found why chrome/chromium versions from finaries does not work, they are compiled with dbus support, we do not have dbus running on the containers, fortunetly the chromium version provided by debian main repositories is not compiled with that option, and it works. So finally, we stop installing chrome from the google nightlys, and we stop downloading the puppeteer chrome version, we replace both by the standard chromium package.

**Pull request merged**